docs/references/configuration_options_reference.md

```diff
@@ -308,6 +309,8 @@ String used to define the configuration of the broker if one is autogenerated. S
 - `--profiler=log` - Send the profiling messages to the default logging file. `log` can be replaced with a path to an alternative file where only the profiling messages will be sent. See the [User Guide page on profiling](../user-guide/advanced_topics/profiling.md) for further details.
 - `--timemonitor=` - Specify the name of the federate to monitor the time from and generate periodic log messages in the broker as the federate updates its time.
 - `--timemonitorperiod=` - can only be used with `--timemonitor`, set the minimum time period which must elapse in simulation before another log message from the time monitor is generated
+- `--logbuffer` - Specify that the last 10 log messages should be stored in a circular buffer to be retrieved via a query
+- `--logbuffersize=` - Specify that the last X log messages should be stored in a circular buffer to be retrieved via a query (only 1 can be used of logbuffersize, and logbuffer)
```

Comment:
Could these be combined into a single `--logbuffer[=size]` option that enables storing log messages in a circular buffer of the size provided (or 10 as a default if just the option with no size is given)?

Reply:
log buffer already does that,  open to changing it if you think that would be better with just a single option.

Reply:
changed it to a single option and updated the documentation and test cases